src/Utils/Helpers.ts

```diff
@@ -36,15 +36,35 @@ export class RealPath {
     this.absPath = absPath;
   }
 
-  public static isEqual(path0: RealPath, path1: RealPath): boolean {
-    return path0.absPath === path1.absPath;
+  public static areEqual(path0: string, path1: string): boolean {
```

Comment:
This function is handy to process `equal` to string paths.

Reply:
I usually seen `is~~~` as `path0 is equal to path1 ?`
I've never seen `are~~~` and at first seeing this, I interpreted this as `path0: string[] are equal to path1: string[] ?`

Reply:
+1 to `isEqual`.

Reply:
I tried to search on a google:

Unity assertion, .NET assertion, C# Selenium, etc,... Uses areEqual. (Google search result : 174,000,000+)
iBatis, Apple uses isEqual, (Apple's for an unary compare operator.) (Google search result: 827,000+)

Reply:
> I've never seen are~~~ and at first seeing this, I interpreted this as path0: string[] are equal to path1: string[] ?

As far as I know, code must be read as it is. In this example,

```areEqual(path0, path1)```

So people will read as: "Are equal: path0 and path1". It sounds gramatically correct.

Whereas this sounds awkward: "Is equal: path0 and path1". 

If it's an unary operator, it isEqual will sound more naturally:
```path0.isEqual(path1)```
"Path0 isEqual to path1"

Reply:
This is your change so you can go as you wish.
But finding examples from internet with `areEqual`, with .NET, I'm not sure this is appropriate.
I've found some examples with TypeScript which I agree upon.
- https://dev.to/macsikora/typescript-is-more-than-you-think-2nbf
- https://www.anycodings.com/questions/how-to-compare-arrays-in-javascript

Anyway, it's up to you.